modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/toolbarbuttons/plugin.js

```diff
@@ -336,6 +336,8 @@
 							this
 						);
 					},
+
+					title: 'Line Spacing',
```

Comment:
I've said it a few time already, I'll repeat : could you **please** consider renaming this to "Line Height"?

Reply:
I was working trying to make things work before fixing these strings. It also needs to be used as `editor.lang.blah`. I'll left a reminder in GIF style for it

![](https://media2.giphy.com/media/2rKDe8JpAEatWVamEe/giphy.gif)